ext/nnstreamer/tensor_filter/vivante/README.md

```diff
@@ -0,0 +1,166 @@
+
+This describes how to enable the NNstreamer tensor filter as a sub-plugin for Vivante NPU.
+
+## Vivante tensor filter
+The entire procedure is as follows:
+ * Read multi-files (e.g., vivante_v3.nb and libvivante_v3.so)
+ * Create a neural network
+ * Execute pre-process for the image data
+ * Verify a graph
+ * Process the graph
+ * Dump all node outputs (optionally, for a debugging)
+ * Do post-process for output data
+ * Release the neural network
+
+## Reference
+ * http://www.vivantecorp.com/
+ * https://www.khadas.com/product-page/vim3 (Amlogic A311D with 5.0 TOPS NPU)
+ 
+## How to buid
+First of all, you must generate a library of a model (e.g., libvivantev3.so, libyolov3.so) to use NNStreamer tensor filter.
+For more details, please refer to the below repository.
+ * https://www.github.com/nnstreamer/vivante-nnstreamer-filter (TODO) - Press 'models' folder.
```

Comment:
I think the address is planned to be changed to `https://github.com/nnstreamer/reference-models`

Reply:
Nice catch. Fixed. :)